tests/Query/Filter/DescriptionField.test.ts

```diff
@@ -15,6 +15,49 @@ function testDescriptionFilter(
     testTaskFilter(filter, task, expected);
 }
 
+declare global {
```

Comment:
WebStorm is showing a number of formatting issues in this code - line lengths too long, missing commas, that sort of thing.

It's not yet showing up in the CI because of the regex failure, but I think it will show up there later.

I see now that you didn't use the PR template, so haven't said which things you checked when developing this.
https://raw.githubusercontent.com/obsidian-tasks-group/obsidian-tasks/main/.github/pull_request_template.md

It would be worth reviewing, please.

Reply:
Running `yarn lint` fixed these. Pushed.